### PR TITLE
#455 change geoobject shaders and handler for fixed visibility feature

### PR DESCRIPTION
**Pull request by pavletto**
Fix visibility function in geo object

- [x] `npm test` passes
- [x] `npm run api` passes
- [x] `npm run lint` passes
- [x] `npm run build` still works

### CODE REVIEW

**Comment by Zemledelec** [comment]
lint?

**Comment by pavletto** [comment]
It's every time red in my machine

File: src/og/shaders/geoObject.js [hunk 1]
```diff
@@ -91,7 +95,7 @@ export const geo_object = () =>
 
                 vPosition = viewMatrixRTE * vec4(highDiff + lowDiff, 1.0);
 
-                gl_Position = projectionMatrix * vPosition;
+                gl_Position = mix(vec4(0.0, 0.0, 0.0, 0.0), projectionMatrix * vPosition, aDispose);
```
Comment (Zemledelec) [inline src/og/shaders/geoObject.js:98]: Maybe better? `projectionMatrix * vPosition * aDispose`

File: src/og/shaders/geoObject.js [hunk 2]
```diff
@@ -200,12 +208,13 @@ export const geo_object_picking = () =>
...
                 viewMatrixRTE[3] = vec4(0.0, 0.0, 0.0, 1.0);
 
-                gl_Position = projectionMatrix *viewMatrixRTE * vec4(highDiff + lowDiff, 1.0);
+                gl_Position = mix(vec4(0.0, 0.0, 0.0, 0.0),  projectionMatrix *viewMatrixRTE * vec4(highDiff + lowDiff, 1.0), aDispose);
             }`,
         fragmentShader:
             `precision highp float;
             varying vec3 vColor;
+            varying float vDispose;
             void main () {
```
Comment (Zemledelec) [inline src/og/shaders/geoObject.js:217]: I'd recommend do not use opacity. Instead use integer type and if with discard.

**Review by Zemledelec — APPROVED** [review]

**Pull request merged**